test-resources-build-tools/src/main/java/io/micronaut/testresources/buildtools/ServerUtils.java

```diff
@@ -57,6 +57,11 @@
 public class ServerUtils {
     public static final String PROPERTIES_FILE_NAME = "test-resources.properties";
 
+    /**
+     * Used only in tests to skip actual port checking
```

Comment:
You're welcome, CheckStyle.

```suggestion
     * Used only in tests to skip actual port checking.
```